Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -523,10 +525,12 @@ def alerts_to_incidents_and_fetch_start_from(alerts, fetch_start_time):
             'occurred': incident_occurred[0] + 'Z',
             'rawJSON': json.dumps(alert)
         }
-        incidents.append(incident)
+        new_last_fetch_id = alert['_id']
+        if alert['_id'] != last_fetched_id:
```

Comment:
```suggestion
        if new_last_fetch_id != last_fetched_id:
```

Reply:
Also, please explain this logic

Reply:
done, Check that the last incident of the previous fetch will not be on the list of the current fetch